yarn-project/noir-contracts/src/contracts/uniswap_contract/src/main.nr

```diff
@@ -124,326 +124,37 @@ contract Uniswap {
-        hash_bytes[262] = canceller_bytes[2];
-        hash_bytes[263] = canceller_bytes[3];
-        hash_bytes[264] = canceller_bytes[4];
-        hash_bytes[265] = canceller_bytes[5];
-        hash_bytes[266] = canceller_bytes[6];
-        hash_bytes[267] = canceller_bytes[7];
-        hash_bytes[268] = canceller_bytes[8];
-        hash_bytes[269] = canceller_bytes[9];
-        hash_bytes[270] = canceller_bytes[10];
-        hash_bytes[271] = canceller_bytes[11];
-        hash_bytes[272] = canceller_bytes[12];
-        hash_bytes[273] = canceller_bytes[13];
-        hash_bytes[274] = canceller_bytes[14];
-        hash_bytes[275] = canceller_bytes[15];
-        hash_bytes[276] = canceller_bytes[16];
-        hash_bytes[277] = canceller_bytes[17];
-        hash_bytes[278] = canceller_bytes[18];
-        hash_bytes[279] = canceller_bytes[19];
-        hash_bytes[280] = canceller_bytes[20];
-        hash_bytes[281] = canceller_bytes[21];
-        hash_bytes[282] = canceller_bytes[22];
-        hash_bytes[283] = canceller_bytes[23];
-        hash_bytes[284] = canceller_bytes[24];
-        hash_bytes[285] = canceller_bytes[25];
-        hash_bytes[286] = canceller_bytes[26];
-        hash_bytes[287] = canceller_bytes[27];
-        hash_bytes[288] = canceller_bytes[28];
-        hash_bytes[289] = canceller_bytes[29];
-        hash_bytes[290] = canceller_bytes[30];
-        hash_bytes[291] = canceller_bytes[31];
-        hash_bytes[292] = callerOnL1_bytes[0];
-        hash_bytes[293] = callerOnL1_bytes[1];
-        hash_bytes[294] = callerOnL1_bytes[2];
-        hash_bytes[295] = callerOnL1_bytes[3];
-        hash_bytes[296] = callerOnL1_bytes[4];
-        hash_bytes[297] = callerOnL1_bytes[5];
-        hash_bytes[298] = callerOnL1_bytes[6];
-        hash_bytes[299] = callerOnL1_bytes[7];
-        hash_bytes[300] = callerOnL1_bytes[8];
-        hash_bytes[301] = callerOnL1_bytes[9];
-        hash_bytes[302] = callerOnL1_bytes[10];
-        hash_bytes[303] = callerOnL1_bytes[11];
-        hash_bytes[304] = callerOnL1_bytes[12];
-        hash_bytes[305] = callerOnL1_bytes[13];
-        hash_bytes[306] = callerOnL1_bytes[14];
-        hash_bytes[307] = callerOnL1_bytes[15];
-        hash_bytes[308] = callerOnL1_bytes[16];
-        hash_bytes[309] = callerOnL1_bytes[17];
-        hash_bytes[310] = callerOnL1_bytes[18];
-        hash_bytes[311] = callerOnL1_bytes[19];
-        hash_bytes[312] = callerOnL1_bytes[20];
-        hash_bytes[313] = callerOnL1_bytes[21];
-        hash_bytes[314] = callerOnL1_bytes[22];
-        hash_bytes[315] = callerOnL1_bytes[23];
-        hash_bytes[316] = callerOnL1_bytes[24];
-        hash_bytes[317] = callerOnL1_bytes[25];
-        hash_bytes[318] = callerOnL1_bytes[26];
-        hash_bytes[319] = callerOnL1_bytes[27];
-        hash_bytes[320] = callerOnL1_bytes[28];
-        hash_bytes[321] = callerOnL1_bytes[29];
-        hash_bytes[322] = callerOnL1_bytes[30];
-        hash_bytes[323] = callerOnL1_bytes[31];
+
```

Comment:
@iAmMichaelConnor

Reply:
As above, they don't need individual loops all of them as it is the same range 🤷